src/App.jsx

```diff
@@ -94,11 +94,11 @@ function App() {
 
   return (
     <div className="App flex flex-col h-full pt-24 min-h-screen dark:text-white w-screen pl-3 pr-3 pb-3 text-zinc-800 bg-zinc-100 dark:bg-zinc-800 overflow-hidden">
-      <LatLongContext.Provider value={contextLatLong}>
-        <PermissionContext.Provider
+      <LatLongContext value={contextLatLong}>
```

Comment:
Is this change related to the React upgrade? I'm just asking out of curiosity.

Reply:
[Yes](https://react.dev/blog/2024/12/05/react-19#context-as-a-provider)